### ISSUE TITLE
export wordlists to a format supported by quizlet

### ISSUE THREAD

**Issue by balmas** [alpheios-project/alpheios-core#201]
format details tbd

requested by Matteo Romanello

**Comment by irina060981** [comment]
As I could see - it could be done easily
Import could be done this way:

![image](https://user-images.githubusercontent.com/12377640/78740030-f758c000-7998-11ea-95ba-c217a50562da.png)

**Comment by irina060981** [comment]
![image](https://user-images.githubusercontent.com/12377640/78745214-d1d2b300-79a6-11ea-8b70-83813801d0a5.png)

In quizlet it would be imported without additional tunning

![image](https://user-images.githubusercontent.com/12377640/78745249-e7e07380-79a6-11ea-9c98-53bc342389dd.png)

**Comment by irina060981** [comment]
Done with minimal requirements from #200 
```
word	definition
beatum (beatus, beata, beatior, beatissimus)	happy, fortunate, bringing happiness; rich, wealthy, copious, sumptuous;
beatum (beatus, beata, beatum)	blessed, blissful; "Saint" (in early Church, less formal);
beatum (beo, beare, beavi, beatus)	bless, make happy, gladden, delight; enrich (with);
beatum (beatus, beati)	happy/fortunate men/persons (pl.); "the_rich"; The Blessed, Saints;
beatum (beatum, beati)	happiness, blessedness; good fortune; good circumstances;
mare (mare, maris)	sea; sea water;
mare (mas, maris)	male (human/animal/plant); man;
mare (mas, (gen.), maris)	male; masculine, of the male sex; manly, virile, brave, noble; G:masculine;
mare (mare)	male; masculine, of the male sex; manly, virile, brave, noble; G:masculine;
```

**Comment by irina060981** [comment]
I suggest to close it and continue working on new requirements inside #200 
@balmas , what do you think ?

**Comment by irina060981** [comment]
Changes are merged

**Comment by balmas** [comment]
> I suggest to close it and continue working on new requirements inside #200
> @balmas , what do you think ?

agree

**Comment by balmas** [comment]
@monzug FYI, the basics for this are in Alpheios Components 3.3.0-qa.20200410462

It's not a finished feature though so wait to test until #200 is done.

**Comment by balmas** [comment]
ok, ready to test in Alpheios Components 3.3.0-qa.20200501441

you need to update the wordlist endpoint in your env-webext.js to test:

    'wordlist' : 'https://userapis-dev.alpheios.net/v1/words',

**Comment by monzug** [comment]
I repeat here an issue reported in #291.
1) limit to important
2) download the wordlist with filtering and with flashcard checked
3) the words in wordlist are still in green as important but the icon has changed in current session.
I believe this should be fixed.
see screenshot
![mark-as-current-session](https://user-images.githubusercontent.com/41396793/82908038-684a3c00-9f35-11ea-80af-d5ca2eeb6050.JPG)

**Comment by balmas** [comment]
>the words in wordlist are still in green as important but the icon has changed in current session (and also limit to).

Do you mean they previously weren't marked as "current session" but now they are? @irina060981 , to fix this, I think when we lookup a word for download, we would have to make that the current session flag doesn't get changed. nor the update date. Maybe we need a different event the wordlist can catch when a lookup is for wordlist download? so that it doesn't affect frequency, updateDate and currentSession?

**Comment by balmas** [comment]
>(and also limit to).

changing value in the filtering sounds like a separate bug

**Comment by irina060981** [comment]
>>the words in wordlist are still in green as important but the icon has changed in current session (and also limit to).

>Do you mean they previously weren't marked as "current session" but now they are? @irina060981 , to fix this, I think when we lookup a word for download, we would have to make that the current session flag doesn't get changed. nor the update date. Maybe we need a different event the wordlist can catch when a lookup is for wordlist download? so that it doesn't affect frequency, updateDate and currentSession?

Yes, I could see the problem, I didn't thought about this with events
I believe we could simply pass additional flag to the event arguments

**Comment by irina060981** [comment]
>>(and also limit to).
>changing value in the filtering sounds like a separate bug

I didn't finally understand - what changed - I made several tests -
  I didn't see that filter field changed
  But I see that results of the filtering could be changed because of updating the words 

@monzug , may be you described changes in filter results?

**Comment by balmas** [comment]
> create specific events for Wordlist updating and for regular lookup
add source parameter to event arguments, define it as a simple string (or may be some constants in data-models) and update events for Homonym_Ready with new format of arguments

I think maybe the use of specific events for Wordlist updating would be the least invasive way to go. It might also help with the issue described in #398

**Comment by irina060981** [comment]
merged

**Comment by irina060981** [comment]
@monzug and @balmas ,
I have here a notice with some strange (limit to) behavior
>the words in wordlist are still in green as important but the icon has changed in current session (and also limit to).

I am not sure what exactly should I test and fix here?
Could you give me some additional explanation?

**Comment by monzug** [comment]
Irina, here is an other screenshot
![limit-to-important](https://user-images.githubusercontent.com/41396793/83520404-a1415e00-a4ab-11ea-89f6-b1c919892eb1.JPG)

when I apply the Limit to important for words saved long time ago, the status of these words is added in current session, which is not the case. the limit to does not change to limit to this session. I applied the limit to this session to see if the words were added there and they are.
